patches/nan/remove_deprecated_v8_isolate_idlenotificationdeadline.patch

```diff
@@ -0,0 +1,28 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: John Kleinschmidt <jkleinsc@electronjs.org>
+Date: Thu, 6 Jun 2024 15:16:49 -0400
+Subject: Remove deprecated v8::Isolate::IdleNotificationDeadline
+
+See https://chromium-review.googlesource.com/c/v8/v8/+/5539852
+
+Also https://github.com/nodejs/nan/issues/953#issuecomment-1791163429
+
+diff --git a/nan.h b/nan.h
```

Comment:
We do have a soft policy for needing "This patch can be removed when...." sections to all new patches, but I guess it's self-evident for these two patches :man_shrugging: